fix(curriculum): add meta test to travel agency page (#58481)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md

1. You should have a `DOCTYPE` declaration.
1. You should have an `html` element with `lang` set to `en`.
1. You should have a `head` element containing a `meta` void element with `charset` set to `utf-8` and a `title` with the text `Travel Agency Page`.

> 1.You should have a`meta`tag in your `head` element that contains a short description of your website for SEO.

Reviewer: ```suggestion
1.  You should have a`meta` tag in your `head` element that contains a short description of your website for SEO.
```

> 1. You should have a`meta` tag in your `head` element that contains a short description of your website for SEO.

Reviewer 2: ```suggestion
1. You should have a `meta` tag in your `head` element that contains a short description of your website for SEO.
```

);
```

You should have two `meta` elements within your `head` element.

```js
assert.match(code, /<\s*head\s*>[\s\S]*(<\s*meta[\s\S]*>[\s\S]*){2,}<\/\s*head\s*>/i);
```

> you should  one `meta` element has name="description" and a non-empty content attribute

Reviewer 2: ```suggestion
One `meta` element should have a `name` attribute with value of `description` and a non-empty `content` attribute.
```

 ```js
 assert.match(code,
  /<\s*meta\s+[^>]*name\s*=\s*["']description["'][^>]*content\s*=\s*["'][^"']+["'][^>]*>/i,
);
```

> Your `meta` element should have its `charset` attribute set to `UTF-8`.

Reviewer 2: ```suggestion
One `meta` element should have its `charset` attribute set to `UTF-8`.
```

);
```

You should have two `meta` elements within your `head` element.

```js

> assert.match(code, /<\s*head\s*>[\s\S]*(<\s*meta[\s\S]*>[\s\S]*){2,}<\/\s*head\s*>/i);

Reviewer 2: this will not work if the `title` or other element is between the two `meta`, right?

```

One `meta` element should have a `name` attribute with value of `description` and a non-empty `content` attribute.

 ```js
 assert.match(code,

>   /<\s*meta\s+[^>]*name\s*=\s*["']description["'][^>]*content\s*=\s*["'][^"']+["'][^>]*>/i,

Reviewer 2: ```suggestion
  /<\s*meta\s+[^>]*name\s*=\s*["']description["'][^>]*content\s*=\s*["'][^"']+?["'][^>]*>/i,
```

better make it lazy before it matches other tags

);
```

> One  `meta` element should have its `charset` attribute set to `UTF-8`.

Reviewer 3: ```suggestion
One `meta` element should have its `charset` attribute set to `UTF-8`.
```
